src/hooks/usePublish.ts

```diff
@@ -83,7 +83,7 @@ function usePublish(): UsePublish {
           break
         }
         case 'compute': {
-          if (!timeout) timeout = 3600
+          if (!timeout) timeout = 0
```

Comment:
I'm not sure why you are changing the timeout to 0 in the case where there isn't a timeout value? To my understanding, 0 will default to forever?

Reply:
Yes, exactly, as it's also suggested in: https://github.com/oceanprotocol/market/blob/b398106238b2e805a7209ffa37218762a4e7f36d/src/utils/metadata.ts#L20 
But you're right, timeout should be equaled to 0  as 'Forever' only for datasets with download asset type, as Alex suggested in the grooming session. I'll revert the change in a moment.